### PR TITLE
Model serializer

### PR DESCRIPTION
**Pull request by martinhath**
Note the last commit, and the change to `test/models.py`. Also, I've never used tox, so im not sure if I did it right.

### CODE REVIEW

**Comment by frecar** [comment]
Can you write some docs as well? :)

File: tox.ini [hunk 1]
```diff
@@ -16,6 +16,7 @@ commands =
 deps =
     coverage
     mock
+    rest_framework
```
Comment (relekang) [inline tox.ini]: I think this should be `djangorestframework`

Comment (martinhath) [inline tox.ini]: Ah, yes.

**Comment by frecar** [comment]
Please make the PR build successfully and fix flake8 issues! :)

**Comment by relekang** [comment]
LGTM :+1:

**Pull request merged**